py/bodynoline.py

```diff
@@ -1,7 +1,7 @@
 import dominate
 from dominate.tags import *
 
-def _bodynoline():
+def _bodynoline(title, entries):
     """Returns a body section with no vertical lines. TODO: edit this description
```

Comment:
Add a description on the params here please: :)

Reply:
added